[KB-183 ] Introduction for model overview and definition for PLS (#132)
chhoumann/thesis-chemcam: report_thesis/src/sections/background.tex

The covariance matrix of the centered data is then computed:

$$
\mathbf{C} = \frac{1}{n-1} \mathbf{\bar{X}}^T \mathbf{\bar{X}},
$$

where $n$ is the number of samples.

Then, the covariance matrix $C$ is decomposed into its eigenvectors $\mathbf{V}$ and eigenvalues $\mathbf{D}$:

$$
\mathbf{C} = \mathbf{V} \mathbf{D} \mathbf{V}^T,
$$

where $\mathbf{V}$ is the matrix of eigenvectors and $\mathbf{D}$ is the diagonal matrix of eigenvalues.

The eigenvectors are then ranked by the magnitude of their corresponding eigenvalues in descending order.
The eigenvectors with the highest eigenvalues are the principal components that capture the most variance in the data.
Usually, only the top $n$ principal components are retained to reduce the dimensionality of the data, as they are assumed to capture the most significant aspects of the data due to their high variance.

Finally, the original data points are projected onto the space defined by the top $n$ principal components, which transforms $X$ into a lower-dimensional representation:

$$
\mathbf{X}_{\text{reduced}} = \mathbf{\bar{X}} \mathbf{V}_n,
$$

where $\mathbf{V}_n$ is the matrix that only contains the top $n$ eigenvectors.

> A property of the principal components is that they are uncorrelated, which means that calculating the covariance matrix of each pair of components results in a diagonal matrix where the off-diagonal elements are zero.

Reviewer: Done - because we already mentioned that they're uncorrelated previously.

\gls{pcr} extends \gls{pca} in the context of regression analysis.
\gls{pcr} involves performing \gls{pca} on the features $\mathbf{X}$ and then using the principal components with the highest variances as features in a linear regression model.
This approach can be beneficial when the original features are highly correlated, as the principal components are orthogonal to each other, reducing multicollinearity.

> If $\mathbf{X}$ is decomposed into $\mathbf{TP}^T + \mathbf{E}$ where $\mathbf{T}$ are the scores and $\mathbf{P}$ are the loadings, \gls{pcr} uses $\mathbf{T}$ to predict the target $\mathbf{y}$ as $\mathbf{y} = \mathbf{Tb} + \mathbf{e}$.

Reviewer 2: What are $\mathbf{b}$ and $\mathbf{e}$ in the regression equation $\mathbf{y} = \mathbf{Tb} + \mathbf{e}$?
Does $\mathbf{e}$ relate to $\mathbf{E}$?

What is the role of the loadings ($\mathbf{P}$) and scores ($\mathbf{T}$) in PCR? They aren't explained, either - unclear what scores & loadings are.

This assumption does not always hold, which is what \gls{pls} aims to address.

\gls{pls} uses an iterative method to identify components that maximize the covariance between the features and the target.m
These components, $Z$, are linear combinations of the original features, $X_j$, weighted by coefficients, $\phi_j$, which are specifically calculated to reflect this covariance.
The formula for each component is expressed as:

$$
    Z = \sum_{j=1}^{p} \phi_j X_j,
$$

where $Z$ represents the component, $X_j$ is the $j$-th feature, and $\phi_j$ is the weight for the $j$-th feature.
The weights, $\phi_j$, are determined by the formula:

$$
    \phi_j = \frac{\text{cov}(X_j, Y)}{\text{var}(X_j)}.
$$

To refine the model iteratively, PLS uses the residuals from the previous components to calculate the next component.
The $m$-th component, for example, is derived from the residuals of the previous $m-1$ components:

$$
    Z_m = \sum_{j=1}^{p} \phi_{jm} \hat{X}_{j, m-1}.
$$

> The components are then used to predict the target variable by fitting a linear model via least squares regression.\cite{James2023AnIS}

Reviewer: OK - but how should we do it then when the entire section is based on a chapter in the book? What's the best way to cite it?

> The components are then used to predict the target variable by fitting a linear model via least squares regression.\cite{James2023AnIS}

Reviewer 2: What about just adding a 'disclaimer' at the top of the PLS section like

> In this section, we provide an overview and definitions of \gls{pls}, primarily based on the methodologies described by \citet{James2023AnIS}.

`Methodologies` isn't the right word, but I couldn't come up with one immediately

It's best to integrate citations more naturally and precisely throughout the text, but I can see how that isn't easy to do.

\mathbf{C} = \mathbf{V} \mathbf{D} \mathbf{V}^T,
$$

> where $\mathbf{V}$, containing the eigenvectors of $\mathbf{C}$, are the principal component loadings.
> These loadings represent the directions of maximum variance in $\mathbf{X}$.
> $\mathbf{D}$, a diagonal matrix, contains the eigenvalues, which quantify the variance captured by each loading.

Reviewer 2: Thoughts? The sentence structure of the last sentence is a bit complex
```suggestion
where matrix $\mathbf{V}$ contains the eigenvectors of $\mathbf{C}$ and represents the principal component loadings.
These loadings indicate the directions of maximum variance in $\mathbf{X}$.
The matrix $\mathbf{D}$ is diagonal and holds the eigenvalues, each of which quantifies the variance captured by its corresponding loading.
```
